contracts/St1inch.sol

```diff
@@ -376,23 +396,44 @@ contract St1inch is ERC20Pods, Ownable, VotingPowerCalculator, IVotable {
 
     // ERC20 methods disablers
 
+     /**
+     * @notice Throws an error indicating that approve functionality is disabled
+     * @dev This function always throws
```

Comment:
It is better to express the idea: "Always reverts to disable token transfer ERC-20 feature .